app/screens/qrcode-scanner-screen/qrcode-scanner-screen.tsx

```diff
@@ -70,14 +70,16 @@ export class QrcodeScannerScreen extends React.Component<QrcodeScannerScreenProp
         const {
           address,
           likerId,
-          coins,
-          memo
+          coins = {},
+          memo,
+          skipToConfirm = true,
         } = JSON.parse(event.data)
         this.props.navigation.replace("Transfer", {
           address,
           likerId,
-          amount: coins?.amount,
+          amount: coins.amount,
```

Comment:
Do we need to fallback to zero instead of undefined?

Reply:
I believe the fallback is set in https://github.com/likecoin/likecoin-app/pull/133/files#diff-5eb0219c91df1602a6fa2c816bd9722cR79